src/app/app.component.spec.ts

```diff
@@ -14,12 +14,6 @@ describe('AppComponent', () => {
     expect(app).toBeTruthy()
   })
 
-  it(`should have as title 'Biomagnethic-App'`, () => {
-    const fixture = TestBed.createComponent(AppComponent)
-    const app = fixture.componentInstance
-    expect(app.title).toEqual('Biomagnethic-App')
-  })
-
   it('should render title', () => {
     const fixture = TestBed.createComponent(AppComponent)
```

Comment:
please stop doing changes to `*.spec.ts` files, we will fix them all at the end

Reply:
restored